api/db/user.js

```diff
@@ -7,7 +9,9 @@ module.exports = {
     },
 
     async activities() {
-      if (!this.relations.role) {
+      logger.silly('getting user activities');
+      if (!this.relations.role || !this.relations.role.activities) {
```

Comment:
Fixed.  Needed to actually check that both of these exist, not just the higher-level one.